Assets/Scripts/UI/MainMenuScreen/Buttons/NewGame.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class NewGame : MonoBehaviour
+{
+    [SerializeField] private MainMenuScreen MainMenuScreenCanvasGroups;
```

Comment:
I don't think we really need a fade. Can delete this and the rest of the commented code.